packages/merge-styles/src/IRawStyleBase.ts

```diff
@@ -739,6 +739,21 @@ export interface IRawStyleBase extends IRawFontStyle {
    */
   clear?: ICSSRule | string;
 
+  /**
+   * The clip CSS property defines what portion of an element is visible. The clip
+   * property applies only to absolutely positioned elements, that is elements with
+   * position:absolute or position:fixed.
+   */
+
+  clip?: ICSSRule | string;
```

Comment:
MDN says this is deprecated, even though all browsers support it. Maybe we should remove it to encourage use of clip-path?

Reply:
clip-path is not supported by Safari according to MDN and this is a commonly used approach among many frameworks.

Also, the Suggestions.scss `.suggestionsAvailable` class sets the precedent for using this style/approach.